featuretools/computational_backends/calculate_feature_matrix.py

```diff
@@ -620,3 +647,133 @@ def get_next_chunk(cutoff_time, time_variable, num_per_chunk):
+
+            workers = njobs_to_workers(njobs)
+            workers = min(workers, len(chunks))
+            cluster = LocalCluster(n_workers=workers,
+                                   threads_per_worker=1,
+                                   diagnostics_port=diagnostics_port,
+                                   **dask_kwargs)
+
+        client = Client(cluster)
+        # scatter the entityset
+        # denote future with leading underscore
+        start = time.time()
+        es_token = "EntitySet-{}".format(tokenize(entityset))
+        if es_token in client.list_datasets():
+            print("Using EntitySet persisted on the cluster as dataset %s" % (es_token))
+            _es = client.get_dataset(es_token)
+        else:
+            _es = client.scatter([entityset])[0]
+            client.publish_dataset(**{_es.key: _es})
+
+        # save features to a tempfile and scatter it
+        pickled_feats = cloudpickle.dumps(features)
+        _saved_features = client.scatter(pickled_feats)
+        client.replicate([_es, _saved_features])
+        end = time.time()
+        scatter_time = end - start
+        scatter_string = "EntitySet scattered to workers in {:.3f} seconds"
+        print(scatter_string.format(scatter_time))
+
+        # map chunks
+        _chunks = client.map(dask_calculate_chunk,
+                             chunks,
+                             saved_features=_saved_features,
+                             entityset=_es,
+                             approximate=approximate,
+                             training_window=training_window,
+                             profile=False,
+                             verbose=False,
+                             save_progress=save_progress,
+                             no_unapproximated_aggs=no_unapproximated_aggs,
+                             cutoff_df_time_var=cutoff_df_time_var,
+                             target_time=target_time,
+                             pass_columns=pass_columns)
+
+        feature_matrix = []
+        # TODO: handle errored / failed
+        iterator = as_completed(_chunks, with_results=True).batches()
+        if verbose:
+            pbar_str = ("Elapsed: {elapsed} | Remaining: {remaining} | "
+                        "Progress: {l_bar}{bar}| "
+                        "Calculated: {n}/{total} chunks")
+            pbar = make_tqdm_iterator(total=len(_chunks), bar_format=pbar_str)
+        for batch in iterator:
+            for future, result in batch:
+                # gather finished futures
+                feature_matrix.append(result)
+                if verbose:
+                    pbar.update()
+        if verbose:
+            pbar.close()
+    except Exception:
+        raise
+    finally:
+        if isinstance(cluster, LocalCluster):
+            cluster.close()
+        if client is not None:
+            client.close()
+
+    return feature_matrix
+
+
+def dask_calculate_chunk(chunk, saved_features, entityset,
```

Comment:
For the feature list we could check if `features` was a string instead of a list and unpickle the string.  For `PandasBackend` we could make `backend` optional and create it inside `calculate_chunk` if `backend` is `None`